devtools/server/actors/replay/module.js

```diff
@@ -1020,6 +1028,14 @@ function getFunctionLocation(obj) {
   if (!script && obj.isBoundFunction) {
     script = obj.boundTargetFunction.script;
   }
+  if (script && gScripts.getId(script) === 0) {
+    log("Error: Cannot get location of unknown script: " + JSON.stringify({
+      url: script.source.url,
+      startLine: script.startLine,
+      startColumn: script.startColumn,
+    }));
```

Comment:
I don't think we need this logging.  This isn't really an error; only a subset of the sources in the recording process are reported to the linker (the ones associated with the page instead of browser internal stuff), and it's perfectly fine for content associated with the sources we care about to have references to browser internal content, but we should hide information about those browser internal scripts like this change is doing.

Reply:
Sounds good. I wasn't sure if we'd expect any of those to get passed show up anywhere in an object preview so I was trying to be cautious, but in the grand scheme not an issue.